securedrop/sass/_base.sass

```diff
@@ -69,7 +69,10 @@
 
   .sd-button
     font-weight: normal
-    letter-spacing: 0.15em
+    &:not([lang=ar])
```

Comment:
This could be future-proofed like so:

```sass
// we have to use two to make it a list otherwise sass complains
$no_space_locales: ar fa
@each $locale in $no_space_locales
  &:lang(locale)
    letter-spacing: 0em 
```

This works because of the precedence of `lang($locale)` is higher than the default and thus it's overridden.

Reply:
Slick! I like it. I'll test and apply this change prior to merge

Reply:
Hmm, I'm having difficulty using this and preserving the existing letter spacing in the button when using English, so I'm gonna merge this PR as is. Will add your code snippet in an issue for safekeeping for when we add Farsi or another language that uses a connected script

Reply:
Sounds good. It might have "worked" for me because of caching. :/